packages/block-library/src/group/editor.scss

```diff
@@ -1,75 +1,17 @@
 /**
  * Group: All Alignment Settings
  */
-
 .wp-block-group {
-
 	// Ensure not rendering outside the element
 	// as -1px causes overflow-x scrollbars
 	.block-editor-block-list__insertion-point {
 		left: 0;
 		right: 0;
 	}
-
```

Comment:
I'm thinking of adding a global styles to the "body" or the editor wrapper or something for the editors that require these styles (existing  behavior) and wrap all these removed styles by that. WDYT?